ui/app/manager/src/pages/page-users.ts

```diff
@@ -855,8 +857,8 @@ export class PageUsers extends Page<AppStateKeyed> {
                                   .label="${i18next.t("username")}"
                                   .type="${InputType.TEXT}" minLength="3" maxLength="255" 
                                   ?required="${isServiceUser || !this._registrationEmailAsUsername}"
-                                  pattern="[A-Za-z0-9\\-_]+"
-                                  .value="${user.username}" autocomplete="false"
+                                  pattern="[A-Za-z0-9\-_+@.ßçʊ]+"
+                                  .value="${user.username}"
```

Comment:
Was the autocomplete attribute removed on purpose?

Reply:
Probably a result of merging with the latest changes in #1566.
So yes, the `autocomplete` attribute should be set to `false`, I'll commit that right away.